roles/install-maven/tasks/main.yaml

```diff
@@ -0,0 +1,19 @@
+- name: Install Apache Maven {{ maven_version }}
+  shell: |
+      wget "https://archive.apache.org/dist/maven/maven-3/{{ maven_version }}/binaries/apache-maven-{{ maven_version }}-bin.tar.gz"
+      tar xzvf apache-maven-{{ maven_version }}-bin.tar.gz
+  args:
+    executable: /bin/bash
+    chdir: '{{ zuul.project.src_dir }}'
+  environment: '{{ global_env }}'
+
+- name: Set maven env vars
+  set_fact:
+    maven_env:
+      PATH: '{{ ansible_user_dir }}/{{ zuul.project.src_dir }}/apache-maven-{{ maven_version }}/bin:{{ ansible_env.PATH }}'
+  no_log: yes
+
+- name: Merge maven env vars into global env
+  set_fact:
+    global_env: '{{ global_env | combine(maven_env) }}'
+  no_log: yes
```

Comment:
Please add a task to show `mvn` path and version:
```bash
which mvn
mvn --version
```

Reply:
Done